README.md

```diff
@@ -24,36 +24,41 @@ initZKWorker();
 
 That's it for the worker file!
 
-To define a ZK state, you simply need to call `createZKState`. Here is an example:
+You define a ZK State in the following way:
 
 ```ts
+import { ZkAppWorkerClient, createZKAssert, createZKState } from "zk-states";
+
+// replace './zkStatesWorker.ts` with the path to the previously defined web worker
+const worker = new Worker(new URL("./zkStatesWorker.ts", import.meta.url), {
+  type: "module",
+});
+const workerClient = new ZkAppWorkerClient(worker);
+
```

Comment:
Maybe add a comment here explaining what `zkAssert` is and why it takes the worker as input.